docs/source/usage.rst

```diff
@@ -48,6 +48,12 @@ The :attr:`LC.meta` attribute contains information needed to calculate absolute
     lc.meta['host_ebv'] = 0.
     lc.meta['redshift'] = 0.002
 
+In the above example, the phase will be calculated based on the first detection. Alternately, the phase can be 
+calculated from an explosion or peak time by populating the :attr:`LC.meta` attribute with the following keys:
+* refmjd: the reference time in MJD
+* explosion: the explosion time in MJD
+* peaktime: the time of the light curve max in MJD. This can be populated manually or using :meth:`lightcurve.LC.findPeak` method
+
```

Comment:
Maybe move this after the following block, because `calcPhase` hasn't been introduced yet.

Reply:
Should we leave `'explosion'` and `'peaktime'` undocumented? I think if someone wants to manually fill in a date, they should just use `'refmjd'` and keep track of what it is themselves. Specifically `'peaktime'` should be reserved for the output of `findPeak` or it could get confusing. But we could point out that they could use `findPeak` and then run `calcPhase(rdsp=True)` if they want to use that feature.

Reply:
I'm not opposed to this - I didn't know that `explosion` existed until I looked at the code and I don't use peaktime